tools/installers/install_speechbrain.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MAKE=make
+
```

Comment:
@Fhrozen It seems we need to make commit and push in local to launch ci.
Could you remove this part and do it?